src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/PackageLevelToGroupNameConverter.cs

```diff
@@ -8,26 +8,30 @@
 
 namespace NuGet.PackageManagement.UI
 {
-    internal class PackageLevelToGroupNameConverter : IValueConverter
+    internal class PackageLevelToGroupNameConverter : IMultiValueConverter
     {
-        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (value is PackageLevel pkgLevel)
+            if (values != null
+                && values.Length == 3
+                && values[0] is PackageLevel pkgLevel
+                && values[1] is int topLevelCount
+                && values[2] is int transitiveCount)
             {
                 if (pkgLevel == PackageLevel.TopLevel)
                 {
-                    return Resources.PackageLevel_TopLevelPackageHeaderText;
+                    return string.Format(Resources.PackageLevel_TopLevelPackageHeaderText, topLevelCount);
                 }
                 else if (pkgLevel == PackageLevel.Transitive)
                 {
-                    return Resources.PackageLevel_TransitivePackageHeaderText;
+                    return string.Format(Resources.PackageLevel_TransitivePackageHeaderText, transitiveCount);
```

Comment:
CurrentUICulture seems to show what's current VS language:

```suggestion
                    return string.Format(Resources.PackageLevel_TransitivePackageHeaderText, transitiveCount, CultureInfo.CurrentUICulture);
```

This also applies for string format line above.